drivers/tee/optee/optee.c

```diff
@@ -198,6 +218,14 @@ static int msg_param_to_param(struct tee_param *param, unsigned int num_param,
 				tp->c = mtp->u.rmem.shm_ref;
 			}
 
+			break;
+		case OPTEE_MSG_ATTR_TYPE_TMEM_INPUT:
+		case OPTEE_MSG_ATTR_TYPE_TMEM_OUTPUT:
+		case OPTEE_MSG_ATTR_TYPE_TMEM_INOUT:
+			rc = msg_param_to_tmp_mem(tp, attr, mtp);
```

Comment:
NIT: can it have non-zero codes at all? As I can see only 0 is possible

Reply:
Yeah, thanks